src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -717,12 +711,6 @@ private void BuildApplicationInitializerBody(IndentedStringBuilder writer, XamlO
 			if (_isUiAutomationMappingEnabled)
 			{
 				writer.AppendLineIndented("global::Uno.UI.FrameworkElementHelper.IsUiAutomationMappingEnabled = true;");
-
-				if (_isWasm)
-				{
-					// When automation mapping is enabled, remove the element ID from the ToString so test screenshots stay the same.
-					writer.AppendLineIndented("global::Uno.UI.FeatureConfiguration.UIElement.RenderToStringWithId = false;");
```

Comment:
As this was a workaround specifically for tests, this is now in SamplesApp itself.

Reply:
Apps won't have this parameter by default. It's best to keep it here.